Adjust the Lid Parameter on the Rate Limited Flapper, May 16, 2022.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Adjust the Lid Parameter on the Rate Limited Flapper, May 16, 2022
+summary: Signal your support or opposition to Adjust the Lid Parameter on the Rate Limited Flapper
+discussion_link: https://forum.makerdao.com/t/signal-request-change-lid-parameter-on-the-rate-limiter/14688
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+   - Technical
+   - Auctions
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-05-1616:00:00
+end_date: 2022-05-1916:00:00
+---
+# Poll:Adjust the Lid Parameter on the Rate Limited Flapper, May 16, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, May 16 at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should vote 'Abstain'.**
+
+## Review
+
+FLAP auctions refer to the Maker Protocol's System Surplus auctions. These auctions sell off DAI revenue from Stability Fees and Liquidation Fees in exchange for MKR which is then burned.
+
+The Rate Limiter enables a more competitive bidding process between keepers and therefore protects Maker from low bids.
+
+The Lid parameter is the total value of DAI allowed to be sent out to surplus auction at once.
+
+The community can vote in this poll to express support or opposition to the following changes: 
+
+* Adjust the lid parameter from 150,000 DAI to 30,000 DAI limiting the amount of parallel flap auctions to 1.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-change-lid-parameter-on-the-rate-limiter/14688) to help inform your position before voting.
+
+For additional information on the parameters please click [here](https://forum.makerdao.com/t/rate-limited-flapper/13056) and [here](https://forum.makerdao.com/t/rate-limiter-parameter-proposal/13193) .
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* This change will be included in an upcoming Executive Vote as the Protocol Engineering Core Unit's schedule allows. 
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Friday executive passes, then this change will become active in the Maker Protocol after the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay) has expired.
```

Comment:
can you change the GSM link to: https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay